docker-entrypoint.sh

```diff
@@ -248,6 +248,8 @@ if (is_numeric($socket)) {
 $user = getenv('WORDPRESS_DB_USER');
 $pass = getenv('WORDPRESS_DB_PASSWORD');
 $dbName = getenv('WORDPRESS_DB_NAME');
+$charset = !empty(getenv('WORDPRESS_DB_CHARSET')) ? getenv('WORDPRESS_DB_CHARSET') : 'utf8mb4';
+$collate = !empty(getenv('WORDPRESS_DB_COLLATE')) ? getenv('WORDPRESS_DB_COLLATE') : 'utf8mb4_unicode_ci';
```

Comment:
Since we've used `:=` above, these can't really be empty, right?  At most we might need to follow up the `:=` line with an `export ...` line?